design.html

```diff
@@ -0,0 +1,1145 @@
+                          <div class="media-body">
+                            <a class=" h5 font-weight-bold text-body" href="">Ahmaud Arbery's Mother Says George Floyd's Death 'Sent Me Back'</a><br>
+                            <span class="h6">To revive the economy, Joe Biden wants to build his own New Deal with an array of proposals to strengthen the social </span>      
+                          </div>
+                        </div>
+                      </div>
+                      <div class="container py-3 border-bottom">
+                        <div class="media  text-justify">
+                          <img src="./assets/things/middle16.PNG" class="align-self-center mr-3"  alt="John Doe" >
+                          <div class="media-body">
+                            <a class=" h5 font-weight-bold text-body" href="">Ahmaud Arbery's Mother Says George Floyd's Death 'Sent Me Back'</a><br>
+                            <span class="h6">To revive the economy, Joe Biden wants to build his own New Deal with an array of proposals to strengthen the social </span>      
+                          </div>
+                        </div>
+                      </div>
+                      <div class="container py-3 border-bottom">
+                        <div class="media  text-justify">
+                          <img src="./assets/things/middle17.PNG" class="align-self-center mr-3"  alt="John Doe" >
+                          <div class="media-body">
+                            <a class="h5 font-weight-bold text-body" href="">Ahmaud Arbery's Mother Says George Floyd's Death 'Sent Me Back'</a><br>
+                            <span class="h6">To revive the economy, Joe Biden wants to build his own New Deal with an array of proposals to strengthen the social </span>      
+                          </div>
+                        </div>
+                      </div>
+                      <div class="container-fluid py-3">
+                        <div class="media  text-justify">
+                          <img src="./assets/things/middle18.PNG" class="align-self-center mr-3"  alt="John Doe" >
+                          <div class="media-body">
+                            <a class="h5 font-weight-bold text-body" href="">Ahmaud Arbery's Mother Says George Floyd's Death 'Sent Me Back'</a><br>
+                            <span class="h6">To revive the economy, Joe Biden wants to build his own New Deal with an array of proposals to strengthen the social </span>      
+                          </div>
+                        </div>
+                      </div>
+                    </div> 
+              <section class="container-fluid w-100   col-lg-3 custom-sticky2 main-divisions order-lg-3 m-0 "> <h3>yy</h3>
+                <div class="  debate-w align-content-center  text-secondary  font-weight-bold  the-debate-sm row ">
```

Comment:
- [ ] The alignment breaks at the mobile screen size.

![image](https://user-images.githubusercontent.com/61479648/88104400-c29ffb80-cb78-11ea-99fc-7cd91f4370a3.png)